Sources/TuistLoader/Loaders/SetupLoader.swift

```diff
@@ -61,30 +68,20 @@ public class SetupLoader: SetupLoading {
     /// - Throws: An error if any of the commands exit unsuccessfully
     ///           or if there isn't a `Setup.swift` file within the project path.
     public func meet(at path: AbsolutePath) throws {
-        let path = try lookupManifest(from: path)
-        logger.info("Setting up the environment defined in \(path.appending(component: Manifest.setup.fileName(path)).pathString)")
-        let setup = try manifestLoader.loadSetup(at: path)
+        guard let setupPath = setupLocator.locate(at: path) else { throw SetupLoaderError.setupNotFound(path) }
+        logger.info("Setting up the environment defined in \(setupPath).pathString)")
+        
+        let setupParentPath = setupPath.parentDirectory
```

Comment:
`manifestLoader.loadSetup(at: setupParentPath)` expects path do directory that contains `Setup.swift`

my new SetupLocator returns path with `Setup.swift`

Reply:
👍